backend/routes/profile.py

```
@@ -0,0 +1,93 @@
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+UPLOAD_DIR = "uploads"
+os.makedirs(UPLOAD_DIR, exist_ok=True)  # Ensure upload directory exists
+
+
+@router.post("/step1", response_model=UserResponse)
+def complete_profile_step1(
```

Ruff (0.8.2)

Missing return type annotation for public function `complete_profile_step1`

(ANN201)